.gitignore

```diff
@@ -17,3 +17,4 @@ jest/config.json
 jest/test-results
 webpack/proxy.dev.js
 src/resources/raml
+.nvmrc
```

Comment:
Actually, let's remove it. I'll put up a PR to add it.
We don't want this in the `.gitignore`, if we want to add it to the project.